source/site-administrators/studio/user-interface-configuration.rst

```diff
@@ -276,9 +276,12 @@ Here's the Sidebar with the folder ``2021/3`` hidden:
 
 .. _sidebar-widget-icon-colors:
 
-*****************************
-Sidebar Widget Icon/Container
-*****************************
+*********************************
+Navigator Widgets Styling Options
+*********************************
+
+Several styling options are available for the navigator widgets. Both the widget's container element
+and the icon element can receive css classes and targeted base styles, collapsed styles and expanded styles.
```

Comment:
CSS

Reply:
@alhambrav @sumerjabri FYI:
- There's a few other instances around the docs where `css` is not in uppercase.
- I've just noticed that documents are duplicated under `/new-ia`; not sure what the or update/transition protocol is. Right now these two are out of sync.